management/client.go

```diff
@@ -229,6 +229,84 @@ type OIDCBackchannelLogout struct {
 	BackChannelLogoutURLs *[]string `json:"backchannel_logout_urls,omitempty"`
 }
 
+// Addons defines the `addons` settings for a Client.
+type Addons struct {
+	// SAML2 Addon configuration. Set this property to indicate that the Addon should be enabled, all settings are optional.
+	// The first entry in `Callbacks` should be the URL to post the SAML Token to
+	SAML2 *SAML2Addon `json:"samlp,omitempty"`
```

Comment:
```suggestion
	SAML2 *ClientAddonSAML2 `json:"samlp,omitempty"`
```

Should we rename this as well to this? 🤔